packages/gobject-introspection/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gi.readthedocs.io/
+TERMUX_PKG_DESCRIPTION="Uniform machine readable API"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.68.0
+TERMUX_PKG_SRCURL=https://github.com/GNOME/gobject-introspection/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=13595a257df7d0b71b002ec115f1faafd3295c9516f307e2c57bd219d5cd8369
+TERMUX_PKG_BUILD_DEPENDS="glib, python"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="-Dincludedir=$TERMUX_PREFIX/python3.9"
```

Comment:
Why pointing to `$TERMUX_PREFIX/python3.9`? This directory doesn't exist, normally...